src/array/cpu/spmat_op_impl_coo.cc

```diff
@@ -708,7 +708,15 @@ inline COOToCSRAlg WhichCOOToCSR(const COOMatrix &coo) {
   if (coo.row_sorted) {
     return COOToCSRAlg::alSorted;
   } else {
+#ifdef _WIN32
+    // On Windows omp_get_max_threads() gives larger value than later OMP can
+    // spawn.
+    int64_t num_threads;
+#pragma omp parallel
+    { num_threads = omp_get_num_threads(); }
```

Comment:
Maybe only master should write this variable?

Reply:
Good point, I will fix it, thanks